game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/seers/seers_stairs.plugin.kts

```diff
@@ -0,0 +1,18 @@
+package gg.rsmod.plugins.content.areas.seers
+
+val stairs = arrayOf(
```

Comment:
You can (and should) use `listOf` here. Arrays are mutable, whereas Lists are immutable, and the data here is intended to be immutable once loaded.

Reply:
fixed